### PR TITLE
gcp/redirect: Bump nginx tag

### PR DESCRIPTION
**Pull request by ameukam**
Changelog: https://nginx.org/en/CHANGES-1.26

### CODE REVIEW

**Comment by ameukam** [comment]
cc @BenTheElder @dims @upodroid

File: apps/k8s-io/deployment.yaml [hunk 1]
```diff
@@ -34,7 +34,7 @@ spec:
           # Map all keys to files.
       containers:
       - name: nginx
-        image: nginx:1.25-alpine
+        image: nginx:1.26-alpine
```
Comment (BenTheElder) [inline apps/k8s-io/deployment.yaml:37]: can we please at least pin the digest?

this is a bit terrifying in the critical path to e.g. binary downloads

Comment (ameukam) [reply apps/k8s-io/deployment.yaml]: Updated. PTAL.

**Comment by BenTheElder** [comment]
/lgtm
/approve
thanks!

**Pull request merged**